spec.html

```diff
@@ -320,7 +320,7 @@ <h1>built-in object</h1>
     </emu-clause>
 
     <emu-clause id="sec-undefined-value">
-      <h1>undefined value</h1>
+      <h1>*undefined* value</h1>
```

Comment:
this kind of feels circular - `*undefined*` is the "undefined value", and vice versa.

(same below with "null value")

Reply:
Doesn't matter much to me. I was just including everything I could find. Happy to drop it if that's not what we want. Actual presentation is identical, anyway (though markup is different).

Reply:
I would very weakly lean towards leaving these without stars, I guess, given that it doesn't affect the market.

Reply:
Done.